packages/search/src/content/index.ts

```diff
@@ -322,30 +326,34 @@ function performEnterpriseContentSearch(
   );
 }
 
+/**
+ * Search datasets from the Hub API (v3) using the same arguments as searchContent()
+ *
+ * NOTE: invalid parameters like isPortal will be ignored
+ * and this returns the Hub API's raw JSONAPI response
+ *
+ * @param request - see searchContent()
+ * @returns Hub API's JSONAPI response
+ */
+export function searchDatasets(
+  request: IContentSearchRequest
+): Promise<{ data: DatasetResource[] }> {
+  const params: ISearchParams = convertToHubParams(request);
+  const requestOptions = { ...getHubRequestOptions(request.options), params };
+  return fetchDatasets(requestOptions);
+}
+
 function performHubContentSearch(
-  request: IContentSearchRequest,
-  defaultPortal?: string,
-  defaultAuthentication?: UserSession
```

Comment:
@drspacemanphd I got rid of these optional arguments in favor of merging them into the `request.options`

Reply:
I believe they were only there for overrides, particularly when invoking `next()` for the next page of results.